orchestrator-salt/src/main/resources/salt/salt/cloudera/agent/start.sls

```diff
@@ -6,6 +6,13 @@ check_token:
     - name: /etc/cloudera-scm-agent/cmagent.token
     - retry:
         attempts: 10
+
+/etc/cloudera-scm-agent/cmagent.token:
+  file.managed:
```

Comment:
shall we add a `require` for the previous state?

Reply:
I can add if you want, but I think it will fail at that point and does not proceed with executing the following states.

Reply:
I'm not sure about that without failhard. I would like to avoid the case when the `check_token` state fails, but this one creates the file and next time it would succeed.
```
If source is left blank or None (use ~ in YAML), the file will be created as an empty file and the content will not be managed.

Reply:
I added a required section
```
----------
          ID: /etc/cloudera-scm-agent/cmagent.token
    Function: file.managed
      Result: False
     Comment: One or more requisite failed: cloudera.agent.start.check_token
     Started: 15:37:03.777529
    Duration: 0.004 ms
     Changes:
----------
          ID: start_agent
    Function: service.running
        Name: cloudera-scm-agent
      Result: False
     Comment: One or more requisite failed: cloudera.agent.start.check_token
     Started: 15:37:03.778363
    Duration: 0.002 ms
     Changes:
```